### PR TITLE
Enable configuring dispatch router logging

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: manifests/router/log.pp [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# $level::          Logging level to use (e.g., debug+, info+)
+#
+# $timestamp::      Enable timestamps
+#
+# $output::         Log file location
+#
+define qpid::router::log(
+  $module         = 'DEFAULT',
+  $level          = 'info+',
+  $timestamp      = true,
```
Comment (ekohl) [inline manifests/router/log.pp]: Use `validate_bool`?

File: manifests/router/log.pp [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  $output         = '/var/log/qdrouterd.log',
```
Comment (ekohl) [inline manifests/router/log.pp]: Use `validate_absolute_path`?

File: manifests/router/log.pp [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+#
+# Configure qpid router logging
+#
+# == Parameters
+#
+# $module::         Logging module to use
+#
+# $level::          Logging level to use (e.g., debug+, info+)
+#
+# $timestamp::      Enable timestamps
```
Comment (ekohl) [inline manifests/router/log.pp]: Add `type:boolean`?

**Comment by stbenjam** [comment]
@ekohl Thanks, updated!

**Comment by ekohl** [comment]
ACK

**Comment by stbenjam** [comment]
Thanks!

**Pull request merged**